### PR TITLE
feat: add iconsax icon

### LINKED ISSUE #2091: [Add] iconsax resource link
### Site URL

https://iconsax-react.pages.dev/

### Description of the above provided site

ICONSAX for React and React Native

### Category

Frontend

### Sub Category

icons

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (optional)
- [X] I am participating in Hacktoberfest 2023

### PR DESCRIPTION
**Pull request by jeeehaan**
Closes #2091

### CODE REVIEW

**Review by Anmol-Baranwal — DISMISSED** [review]
🚀 Looks good to me.

File: database/frontend/icons.json [hunk 1]
```diff
@@ -243,5 +243,12 @@
     "url": "https://react-icons.github.io/react-icons/",
     "category": "frontend",
     "subcategory": "icons"
+  },
+  {
+    "name": "Iconsax",
+    "description": "ICONSAX for React and React Native.",
```
Comment (rupali-codes) [inline database/frontend/icons.json:249]:
```suggestion
    "description": "ICONSAX provides a set of icons to use in React & React Native projects.",
```

**Review by Anmol-Baranwal — APPROVED** [review]
🚀 Looks good to me.

**Review by aftabrehan — APPROVED** [review]
Looks good to me! 👍

**Review by rupali-codes — APPROVED** [review]
lgtm

**Pull request merged**